exercises/third_task.go/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "fmt"
+
+func append2(x *[]int, y *[]int) { *x = append(*x, *y...) }
```

Comment:
nit: `y` could have been just a slice (not a pointer), or you could also use ellipsis syntax:
```go
func append2(x *[]int, y...int) { *x = append(*x, y...) }
```
`y` is `[]int` in this context (-; but allows to call the function with variable amount of args: `append2(&x, 1, 2, 3, 4)`